agent/fw_drupal8.c

```diff
@@ -532,6 +533,22 @@ NR_PHP_WRAPPER(nr_drupal8_name_the_wt_via_symfony) {
 }
 NR_PHP_WRAPPER_END
 
+void nr_drupal_version() {
+  char* string = "Drupal::VERSION;";
```

Comment:
Since this value will not change, consider explicitly using the `const` keyword. 
The function `zend_eval_string` takes a `const char*` as it's first argument, so it also aligns with the PHP function expectation.

Reply:
After discussion, we will not be adding the `const` keyword here because the `zend_eval_string` function before PHP 8.0 does not include the const keyword in it's first argument.